storybook/src/components/Row/Row.docs.mdx

```diff
@@ -24,6 +24,21 @@ Wrap a Row around a set of components that need the same amount of white space b
 
 <Controls />
 
+### Horizontal Alignment
+
+Items can be aligned horizontally to the start, center or end of the row.
+Or whitespace can be distributed between, around or evenly around them.
```

Comment:
Maybe add here that start alignment is default?

Reply:
The robots even advised to write the options out in a list. It is longer, but they still have a point:

> This revision organizes the information into bullet points, making it easier to scan and understand the options available. It also provides a consistent format for listing the alignment types and their descriptions.